standalone-metastore/src/main/java/org/apache/hadoop/hive/metastore/Warehouse.java

```diff
@@ -58,10 +59,13 @@
  * This class represents a warehouse where data of Hive tables is stored
  */
 public class Warehouse {
+  public static final String DEFAULT_CATALOG_NAME = "hive";
```

Comment:
Wouldn't it better for this to be a configurable property rather then hard-coded value?